website/orders/models.py

```diff
@@ -620,50 +661,16 @@ def __str__(self):
         return f"{self.product} for {self.user} ({self.shift})"
 
     def clean(self):
-        """Check whether this order is valid and can be saved."""
-        super().clean()
-        errors = {}
-        timezone = pytz.timezone(settings.TIME_ZONE)
-        localized_time = timezone.localize(datetime.now())
-        if not self.created and not self.shift.can_order:
-            errors.update({"shift": "You can't order for this shift"})
-
-        if not self.created and not (self.shift.start_date < localized_time < self.shift.end_date):
-            errors.update({"shift": "You can't order for this shift because of time restrictions"})
+        """
+        Clean this Order.
 
-        if (
-            self.shift.max_orders_per_user is not None
-            and Order.objects.filter(shift=self.shift, user=self.pk, product__ignore_shift_restrictions=False)
-            .exclude(pk=self.pk)
-            .count()
-            >= self.shift.max_orders_per_user
-        ):
-            errors.update(
-                {
-                    "shift": f"You are not allowed to order more than {self.shift.max_orders_per_user} "
-                    f"products in this shift."
-                }
-            )
-        if (
-            self.product.max_allowed_per_shift is not None
-            and not self.product.ignore_shift_restrictions
-            and Order.objects.filter(product=self.product, user=self.pk).exclude(pk=self.pk).count()
-            >= self.product.max_allowed_per_shift
-        ):
-            errors.update(
-                {
-                    "product": f"You are not allowed to order more than {self.product.max_allowed_per_shift} "
-                    f"products of this kind in this shift."
-                }
-            )
-
-        if self.type == self.TYPE_ORDERED and (self.user is None or self.user == ""):
-            errors.update(
-                {"user": "A user must be specified if the order type is {}".format(self.TYPES[self.TYPE_ORDERED][1])}
-            )
-
-        if errors:
-            raise ValidationError(errors)
+        Check if the Shift is already finalized.
+        :return:
+        :rtype:
+        """
+        super().clean()
```

Comment:
I don't see how all those cleans you removed have moved to a super class?

Reply:
This is just good coding practice right? Calling the super method?

Reply:
But you're removing all other checks, like the check if you can still order because of time restrictions, max_orders_per_user restrictions, etc